Makefile

```diff
@@ -177,7 +177,7 @@ prove: prove_tests prove_python_tests prove_apps
 prove_tests:
 	$(MAKE) -C tests/spark prove
 
-prove_python_tests: TEST_PROCS=1
+prove_python_tests: export GNATPROVE_PROCS=1
```

Comment:
Do we need `export` here? And is `GNATPROVE_PROCS` already defined somewhere?

Reply:
`GNATPROVE_PROCS` is an environment variable which is read by `gnatprove`, so just setting the variable in the Makefile without `export` is not sufficient. `GNATPROVE_PROCS` is not used in the Makefile elsewhere.

`GNATPROVE_PROCS` defines the maximum number of jobs used by `gnatprove`. As we already start multiple parallel pytest processes, `gnatprove` shall not use multiprocessing in this specific case.